src/Configurator/EnvConfigurator.php

```diff
@@ -47,15 +59,44 @@ public function configure(Recipe $recipe, $vars): void
         file_put_contents(getcwd().'/.env', $data, FILE_APPEND);
     }
 
-    public function unconfigure(Recipe $recipe, $vars): void
+    private function configurePhpUnit(Recipe $recipe, $vars): void
+    {
+        foreach (['phpunit.xml.dist', 'phpunit.xml'] as $file) {
+            $phpunit = getcwd().'/'.$file;
+            if (!is_file($phpunit)) {
+                continue;
+            }
+
+            if ($this->isFileXmlMarked($recipe, $phpunit)) {
+                continue;
+            }
+
+            $data = '';
+            foreach ($vars as $key => $value) {
+                if ('%generate(secret)%' === $value) {
+                    $value = bin2hex(random_bytes(16));
+                }
+                if ('#' === $key[0]) {
+                    $data .= '        <!-- '.$value." -->\n";
+                } else {
+                    $value = $this->options->expandTargetDir($value);
+                    $data .= "        <server name=\"$key\" value=\"$value\" />\n";
```

Comment:
According to https://github.com/symfony/recipes/pull/170 shouldn't be `<env ... />`?

Reply:
indeed, fixed. Thanks.